### DISCUSSION TITLE
TP3 - ejercicio 1 - Promedio del arreglo

### DISCUSSION THREAD

**Discussion by Juampi44** [INGCOM-UNRN-P1/cursada-2023#115] [Preguntas y respuestas] [answered]
puedo definir una funcion como double para sacar el promedio de los numeros del array?

**Comment by martinvilu** [comment, marked as answer]
Si, la pregunta es; ¿de que otra forma la definirías?

**Reply by Juampi44** [reply]
como float?

**Reply by martinvilu** [reply]
¿Y eso estaria mal?

**Reply by martinvilu** [reply]
(te pregunto por qué son buenas preguntas de parcial 😄)

**Reply by Juampi44** [reply]
emmmmmmmm jaja creo que no, porque el float toma numeros decimales, double es mejor porque ocupa menos espacio y es mas preciso

**Reply by martinvilu** [reply]
Ambos representan números decimales, y el double, es el doble de float, en tamaño y en números posibles a representar.
